src/NetPanzer/Core/main.cpp

```diff
@@ -242,12 +242,15 @@ BaseGameManager* initialise(int argc, char** argv) {
     exit(1);
   }
 
-#ifdef NP_DATADIR
-  try {
-    filesystem::addToSearchPath(NP_DATADIR);
-  } catch (...) {
+  Package::assignDataDir();
+  const std::string dataDir = Package::getDataDir();
+  if (dataDir != "") {
```

Comment:
can this be a nullptr?

Reply:
getDataDir returns an std::string. https://www.phind.com/search?q=when+an+std+string+is+declared+in+C+without+a+value%2C+is+it+an+empty+string+or+a+null+pointer

Reply:
ah ok!